test/utils.test.mjs

```diff
@@ -80,6 +80,10 @@ describe('Test Utils', () => {
     assert.equal('desktop:chromeos', getMaskedUserAgent(getUserAgentHeaders('Mozilla/5.0 (X11; CrOS x86_64 14541.0.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36')));
     assert.equal('desktop', getMaskedUserAgent(getUserAgentHeaders('foobar')));
 
+    const m = new Map();
+    m.set('x-newrelic-id', 'VQcPU1ZQGwYEXVZUBwY=');
+    assert.equal('bot:monitoring', getMaskedUserAgent(m));
```

Comment:
You could write this more succinctly as which is a bit more similar to the other assertions in this test:
```suggestion
    assert.equal('bot:monitoring', getMaskedUserAgent(new Map().set('x-newrelic-id', 'VQcPU1ZQGwYEXVZUBwY=')));
```